Fixes #24 Update DCI

Yuri05 opened the pull request:
Fixes #25 Update Simmodel
Fixes #44 Add 64bit version of DCIR

msevestre commented on src/manual/MoBiToolboxForR-package.Rd:
```diff
@@ -12,8 +12,8 @@ This package provides functions to access a MoBi
-Version: \tab 6.3.1\cr
-Date: \tab 2016-07-20\cr
+Version: \tab 7.3.0\cr
+Date: \tab 2018-30-01\cr
```
Could not that be read from a file generated during build? This would be so much nicer than always having to adapt it by hand

Yuri05 replied:
Maybe

msevestre commented:
https://github.com/Open-Systems-Pharmacology/R-Toolbox/blob/4637e664ad6e692e1a59af9f22aacc8e8b7de39d/src/code/initSimulation.R


https://github.com/Open-Systems-Pharmacology/R-Toolbox/blob/d3c8608cd9a9dd3b3d7eac34e31d9d7130013c1c/setup/NAMESPACE

Yuri05 commented:
Moved to VS2017 and added x64-version of DCIR. 

Once this PR is merged, VS2017 Runtime must be installed on the target computer (32 or 64 bit - depending on used R-Toolbox) (@StephanSchaller @PavelBal FYI)

PavelBal commented:
Nice!
